--- settings.yml	2024-11-08 13:17:33.921924400 -0500
+++ "settings - Copy.yml"	2024-11-02 22:27:48.000000000 -0400
@@ -32,15 +32,15 @@
   # Existing autocomplete backends: "dbpedia", "duckduckgo", "google", "yandex", "mwmbl",
   # "seznam", "startpage", "stract", "swisscows", "qwant", "wikipedia" - leave blank to turn it off
   # by default.
-  autocomplete: ""
+  autocomplete: "duckduckgo"
   # minimun characters to type before autocompleter starts
   autocomplete_min: 4
   # backend for the favicon near URL in search results.
   # Available resolvers: "allesedv", "duckduckgo", "google", "yandex" - leave blank to turn it off by default.
-  favicon_resolver: ""
+  favicon_resolver: "duckduckgo"
   # Default search language - leave blank to detect from browser information or
   # use codes from 'languages.py'
-  default_lang: "auto"
+  default_lang: "en-US"
   # max_page: 0  # if engine supports paging, 0 means unlimited numbers of pages
   # Available languages
   # languages:
@@ -73,6 +73,7 @@
   # formats: [html, csv, json, rss]
   formats:
     - html
+    - json
 
 server:
   # Is overwritten by ${SEARXNG_PORT} and ${SEARXNG_BIND_ADDRESS}
@@ -91,7 +92,7 @@
   # If your instance owns a /etc/searxng/settings.yml file, then set the following
   # values there.
 
-  secret_key: "8417622cad8a786f985b40320497e874e26846bdce1afdb4686360a6280b9018"  # Is overwritten by ${SEARXNG_SECRET}
+  secret_key: "5e77fb0401a91c23028a0fd369366f78150c108428e638d19e1700e14aac6646"  # Is overwritten by ${SEARXNG_SECRET}
   # Proxy image results through SearXNG. Is overwritten by ${SEARXNG_IMAGE_PROXY}
   image_proxy: false
   # 1.0 and 1.1 are supported
@@ -138,7 +139,7 @@
     simple_style: auto
   # Perform search immediately if a category selected.
   # Disable to select multiple categories at once and start the search manually.
-  search_on_category_select: true
+  search_on_category_select: false
   # Hotkeys: default or vim
   hotkeys: default
 
@@ -220,7 +221,7 @@
 
 # Comment or un-comment plugin to activate / deactivate by default.
 #
-# enabled_plugins:
+enabled_plugins:
 #   # these plugins are enabled if nothing is configured ..
 #   - 'Basic Calculator'
 #   - 'Hash plugin'
@@ -229,7 +230,7 @@
 #   - 'Ahmia blacklist'  # activation depends on outgoing.using_tor_proxy
 #   # these plugins are disabled if nothing is configured ..
 #   - 'Hostnames plugin'  # see 'hostnames' configuration below
-#   - 'Open Access DOI rewrite'
+  - 'Open Access DOI rewrite'
 #   - 'Tor check plugin'
 #   # Read the docs before activate: auto-detection of the language could be
 #   # detrimental to users expectations / users can activate the plugin in the
@@ -350,7 +351,7 @@
     engine: apkmirror
     timeout: 4.0
     shortcut: apkm
-    disabled: true
+    disabled: false
 
   - name: apple app store
     engine: apple_app_store
@@ -376,7 +377,7 @@
     categories: it
     timeout: 6.0
     shortcut: conda
-    disabled: true
+    disabled: false
 
   - name: arch linux wiki
     engine: archlinux
@@ -406,6 +407,7 @@
     engine: bandcamp
     shortcut: bc
     categories: music
+    disabled: true
 
   - name: wikipedia
     engine: wikipedia
@@ -428,14 +430,17 @@
   - name: bing images
     engine: bing_images
     shortcut: bii
+    disabled: true
 
   - name: bing news
     engine: bing_news
     shortcut: bin
+    disabled: true
 
   - name: bing videos
     engine: bing_videos
     shortcut: biv
+    disabled: true
 
   - name: bitbucket
     engine: xpath
@@ -484,6 +489,7 @@
     shortcut: chef
     # to show premium or plus results too:
     # skip_premium: false
+    disabled: true
 
   - name: cloudflareai
     engine: cloudflareai
@@ -513,13 +519,13 @@
     engine: cppreference
     shortcut: cpp
     paging: false
-    disabled: true
+    disabled: false
 
   - name: crossref
     engine: crossref
     shortcut: cr
     timeout: 30
-    disabled: true
+    disabled: false
 
   - name: crowdview
     engine: json_engine
@@ -687,26 +693,26 @@
     categories: [images, web]
     ddg_category: images
     shortcut: ddi
-    disabled: true
+    disabled: false
 
   - name: duckduckgo videos
     engine: duckduckgo_extra
     categories: [videos, web]
     ddg_category: videos
     shortcut: ddv
-    disabled: true
+    disabled: false
 
   - name: duckduckgo news
     engine: duckduckgo_extra
     categories: [news, web]
     ddg_category: news
     shortcut: ddn
-    disabled: true
+    disabled: false
 
   - name: duckduckgo weather
     engine: duckduckgo_weather
     shortcut: ddw
-    disabled: true
+    disabled: false
 
   - name: apple maps
     engine: apple_maps
@@ -760,7 +766,7 @@
   - name: fdroid
     engine: fdroid
     shortcut: fd
-    disabled: true
+    disabled: false
 
   - name: findthatmeme
     engine: findthatmeme
@@ -776,6 +782,7 @@
     # api_key: 'apikey' # required!
     # Or you can use the html non-stable engine, activated by default
     engine: flickr_noapi
+    disabled: true
 
   - name: free software directory
     engine: mediawiki
@@ -784,7 +791,7 @@
     base_url: https://directory.fsf.org/
     search_type: title
     timeout: 5.0
-    disabled: true
+    disabled: false
     about:
       website: https://directory.fsf.org/
       wikidata_id: Q2470288
@@ -816,6 +823,7 @@
   - name: genius
     engine: genius
     shortcut: gen
+    disabled: true
 
   - name: gentoo
     engine: mediawiki
@@ -830,7 +838,7 @@
     engine: gitlab
     base_url: https://gitlab.com
     shortcut: gl
-    disabled: true
+    disabled: false
     about:
       website: https://gitlab.com/
       wikidata_id: Q16639197
@@ -871,6 +879,7 @@
     shortcut: go
     # additional_tests:
     #   android: *test_android
+    disabled: true
 
   - name: google images
     engine: google_images
@@ -883,18 +892,21 @@
     #       lang: ['en', 'de', 'fr', 'zh-CN']
     #     result_container:
     #       - ['one_title_contains', 'Salvador']
+    disabled: true
 
   - name: google news
     engine: google_news
     shortcut: gon
     # additional_tests:
     #   android: *test_android
+    disabled: true
 
   - name: google videos
     engine: google_videos
     shortcut: gov
     # additional_tests:
     #   android: *test_android
+    disabled: true
 
   - name: google scholar
     engine: google_scholar
@@ -943,7 +955,7 @@
   - name: hackernews
     engine: hackernews
     shortcut: hn
-    disabled: true
+    disabled: false
 
   - name: hex
     engine: hex
@@ -956,7 +968,7 @@
   - name: crates.io
     engine: crates
     shortcut: crates
-    disabled: true
+    disabled: false
     timeout: 6.0
 
   - name: hoogle
@@ -1027,6 +1039,7 @@
       - https://kat.am
     shortcut: kc
     timeout: 4.0
+    disabled: true
 
   - name: lemmy communities
     engine: lemmy
@@ -1075,6 +1088,7 @@
     shortcut: zlib
     categories: files
     timeout: 7.0
+    disabled: true
 
   - name: library of congress
     engine: loc
@@ -1157,7 +1171,7 @@
   - name: metacpan
     engine: metacpan
     shortcut: cpan
-    disabled: true
+    disabled: false
     number_of_results: 20
 
   # - name: meilisearch
@@ -1170,6 +1184,7 @@
   - name: mixcloud
     engine: mixcloud
     shortcut: mc
+    disabled: true
 
   # MongoDB engine
   # Required dependency: pymongo
@@ -1206,7 +1221,7 @@
     engine: npm
     shortcut: npm
     timeout: 5.0
-    disabled: true
+    disabled: false
 
   - name: nyaa
     engine: nyaa
@@ -1287,7 +1302,7 @@
     engine: openlibrary
     shortcut: ol
     timeout: 5
-    disabled: true
+    disabled: false
 
   - name: openmeteo
     engine: open_meteo
@@ -1355,6 +1370,7 @@
   - name: pinterest
     engine: pinterest
     shortcut: pin
+    disabled: true
 
   - name: piped
     engine: piped
@@ -1378,6 +1394,7 @@
     categories: music
     piped_filter: music_songs
     timeout: 3.0
+    disabled: true
 
   - name: piratebay
     engine: piratebay
@@ -1386,6 +1403,7 @@
     # country
     url: https://thepiratebay.org/
     timeout: 3.0
+    disabled: true
 
   - name: pixiv
     shortcut: pv
@@ -1497,6 +1515,7 @@
     categories: [general, web]
     additional_tests:
       rosebud: *test_rosebud
+    disabled: true
 
   - name: qwant news
     qwant_categ: news
@@ -1504,6 +1523,7 @@
     shortcut: qwn
     categories: news
     network: qwant
+    disabled: true
 
   - name: qwant images
     qwant_categ: images
@@ -1511,6 +1531,7 @@
     shortcut: qwi
     categories: [images, web]
     network: qwant
+    disabled: true
 
   - name: qwant videos
     qwant_categ: videos
@@ -1518,6 +1539,7 @@
     shortcut: qwv
     categories: [videos, web]
     network: qwant
+    disabled: true
 
   # - name: library
   #   engine: recoll
@@ -1544,12 +1566,13 @@
   - name: radio browser
     engine: radio_browser
     shortcut: rb
+    disabled: true
 
   - name: reddit
     engine: reddit
     shortcut: re
     page_size: 25
-    disabled: true
+    disabled: false
 
   - name: right dao
     engine: xpath
@@ -1612,6 +1635,7 @@
   - name: soundcloud
     engine: soundcloud
     shortcut: sc
+    disabled: true
 
   - name: stackoverflow
     engine: stackexchange
@@ -1660,7 +1684,7 @@
   - name: searchcode code
     engine: searchcode_code
     shortcut: scc
-    disabled: true
+    disabled: false
 
   # - name: searx
   #   engine: searx_engine
@@ -1705,7 +1729,7 @@
     engine: startpage
     shortcut: sp
     timeout: 6.0
-    disabled: true
+    disabled: false
     additional_tests:
       rosebud: *test_rosebud
 
@@ -1722,6 +1746,7 @@
     base_url:
       - https://solidtorrents.to
       - https://bitsearch.to
+    disabled: true
 
   # For this demo of the sqlite engine download:
   #   https://liste.mediathekview.de/filmliste-v2.db.bz2
@@ -1839,6 +1864,7 @@
   - name: yahoo news
     engine: yahoo_news
     shortcut: yhn
+    disabled: true
 
   - name: youtube
     shortcut: yt
@@ -1879,7 +1905,7 @@
     categories: [general, wikimedia]
     base_url: "https://{language}.wikibooks.org/"
     search_type: text
-    disabled: true
+    disabled: false
     about:
       website: https://www.wikibooks.org/
       wikidata_id: Q367
@@ -2309,6 +2335,7 @@
     engine: wttr
     shortcut: wttr
     timeout: 9.0
+    disabled: true
 
   - name: yummly
     engine: yummly
@@ -2358,7 +2385,7 @@
   - name: lib.rs
     shortcut: lrs
     engine: lib_rs
-    disabled: true
+    disabled: false
 
   - name: sourcehut
     shortcut: srht
